include/versioningCompiler/Utils.hpp

```diff
@@ -81,6 +81,13 @@ void *compileAndGetSymbol(version_ptr_t &v) {
   return nullptr;
 }
 
+std::vector<void *> compileAndGetSymbols(version_ptr_t &v) {
```

Comment:
```suggestion
std::vector<void *> compileAndGetVectorSymbols(version_ptr_t &v) {
```
It is quite evil to create 2 APIs that differ only by a single character